lib/blacklight/models/wikipage.rb

```diff
@@ -8,6 +8,14 @@ def canvas_conversion(course, resources)
           select { |p| p.title.start_with? @title }.count
         @title = "#{@title}-#{page_count + 1}" if page_count > 0
         page = CanvasCc::CanvasCC::Models::Page.new
+        if !@url.empty?
+          @body = %{
```

Comment:
I could be misunderstanding the flow somewhere, but I was thinking of this sort of timeline: 
```
iterate_xml is called on a Content object --> @body is set with some content if it exists --> canvas_conversion is called to create a wiki page
```
With this timeline if you call `@body = ` during canvas_conversion then if there was any sort of description being stored in @body it would be replaced. 

Where am I misunderstanding?